src/data_management/timeseries/locations/LocationForm.tsx

```diff
@@ -389,124 +207,36 @@ const LocationFormModel = (props:Props & PropsFromDispatch & RouteComponentProps
           triedToSubmit={triedToSubmit}
           onFocus={handleFocus}
           onBlur={handleBlur}
-        /> */}
+        />
         <span className={formStyles.FormFieldTitle}>
           3: Rights
         </span>
         <AccessModifier
-          title={'Accessibility'}
+          title={'Accessibility *'}
           name={'accessModifier'}
           value={values.accessModifier}
           valueChanged={value => handleValueChange('accessModifier', value)}
           onFocus={handleFocus}
           onBlur={handleBlur}
         />
-        {/* <SelectDropdown
-          title={'Supplier'}
-          name={'supplier'}
-          placeholder={'- Search and select -'}
-          value={values.supplier}
-          valueChanged={value => handleValueChange('supplier', value)}
-          options={supplierIds.map((suppl:any) => convertToSelectObject(suppl.username))}
-          validated
-          onFocus={handleFocus}
-          onBlur={handleBlur}
-          readOnly={(!(supplierIds.length > 0 && selectedOrganisation.roles.includes('admin')))}
-        /> */}
-        {/* <TextInput
-          title={'Label type Uuid'}
-          name={'uuid'}
-          value={values.uuid}
-          valueChanged={handleInputChange}
-          clearInput={clearInput}
-          validated={!minLength(3, values.uuid)}
-          errorMessage={minLength(3, values.uuid)}
-          triedToSubmit={triedToSubmit}
-          onFocus={handleFocus}
-          onBlur={handleBlur}
-          readOnly={true}
-        />
-        <TextArea
-          title={'Description'}
-          name={'description'}
-          placeholder={'Description here..'}
-          value={values.description}
-          valueChanged={handleInputChange}
-          clearInput={clearInput}
-          validated
-          onFocus={handleFocus}
-          onBlur={handleBlur}
-          readOnly={true}
-        />
-        <TextInput
-          title={'Organisation'}
-          name={'organisation'}
-          value={values.organisation}
-          valueChanged={handleInputChange}
-          clearInput={clearInput}
-          validated
-          onFocus={handleFocus}
-          onBlur={handleBlur}
-          readOnly
-        /> */}
         <div
           className={formStyles.ButtonContainer}
         >
           <CancelButton
             url={'/data_management/timeseries/locations'}
           />
-          <div style={{
-            display: "flex"
-          }}>
-            {currentRecord?
-             <div style={{marginRight: "16px"}}> 
-              <FormActionButtons
-                actions={[
-                  {
-                    displayValue: "Delete",
-                    actionFunction: () => {setShowDeleteModal(true)}
-                  },
-                ]}
-              />
-            </div>
-            :null}
```

Comment:
Removed the FormAction for now since this is also not supported in other forms. We can make a ticket to add the FormAction to all the forms if neccessary.